### PR TITLE
Check Carthage compatibility by `carthage build --no-skip-current`

### PR DESCRIPTION
**Pull request by ishkawa**

### CODE REVIEW

**Comment by ishkawa** [comment]
Pend until https://github.com/Carthage/Carthage/issues/447 is resolved.

File: circle.yml [hunk 1]
```diff
@@ -18,8 +18,16 @@ dependencies:
...
     - pod lib lint
     - set -o pipefail && xcodebuild test -scheme APIKit-iOS | xcpretty -c -r junit -o $CIRCLE_TEST_REPORTS/test-report-ios.xml
     - set -o pipefail && xcodebuild test -scheme APIKit-Mac | xcpretty -c -r junit -o $CIRCLE_TEST_REPORTS/test-report-mac.xml
     - set -o pipefail && xcodebuild build -scheme DemoApp -sdk iphonesimulator | xcpretty -c
+
+deployment:
+  master:
+    branch: master
+    commands:
+      - (cd Carthage; zip -r APIKit.framework.zip Build -i Build/*/APIKit.framework/**/*)
```
Comment (ikesyo) [inline circle.yml]: `carthage archive APIKit` can be used here, I think. :wink:

Comment (ishkawa) [inline circle.yml]: Oh, great. I overlooked it :sweat_smile:

**Comment by ishkawa** [comment]
https://github.com/Carthage/Carthage/issues/447 is resolved and CI build succeeded!

**Comment by ikesyo** [comment]
Looks very good! :shipit:

**Pull request merged**